README.md

```diff
@@ -1,32 +1,34 @@
-# Tempest Graph Library and Database (Beta)
+# Tempest: A real-time graph database (Beta)
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
 - [Features](#features)
+- [Introduction](#introduction)
+  - [Graphs are Everywhere](#graphs-are-everywhere)
   - [Applications of Graph Analysis](#applications-of-graph-analysis)
   - [Why a new Graph Database?](#why-a-new-graph-database)
   - [Our Approach](#our-approach)
-- [Using the Tempest library](#using-the-tempest-library)
-  - [Requirements](#requirements)
-  - [Converting a Graph to Tempest Format](#converting-a-graph-to-tempest-format)
-  - [Using Tempest from Scala](#using-tempest-from-scala)
-  - [Using Tempest from Java](#using-tempest-from-java)
-  - [Using Tempest from Python](#using-tempest-from-python)
+  - [Created by](#created-by)
 - [Using Tempest DB](#using-tempest-db)
   - [Recommended Machine Sizes](#recommended-machine-sizes)
+- [Using Tempest as a library](#using-the-tempest-library)
+  - [Requirements](#requirements)
+  - [Converting a Graph to Tempest Format](#converting-a-graph-to-tempest-format)
+  - [Language Support for Tempest](#language-support-for-tempest)
+    - [Scala](#scala)
+    - [Java](#java)
+    - [Python](#python)
 - [Project Roadmap](#project-roadmap)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Features
-- Scalable: Tempest handles graphs with billions of nodes and edges.  For example, it can store a 30-billion
-  edge graph using 300GB of RAM.
-- Fast:  Accessing a neighbor of a node is as fast as two RAM accesses.   Thousands of edges
-  can be added per second when storing them durably, and hundreds of thousands can be added per 
-  second when storing in RAM only.
-- Convenient: Tempest DB is installed with a single Docker command.  The Tempest library is available
-  as a single sbt or Maven dependency.  
+- Single node, scale-up architecture: Graphs are notoriously hard to partition if you also need to support real-time traversals. So Tempest runs on a single commodity node with large-ish memory (see [here](#recommended-machine-sizes)).
+- Scalable: Despite running on a single Tempest supports graphs with billions of nodes and edges.  For example, it can store a 30 billion edge graph using 300GB of RAM.
```

Comment:
"single Tempest" => "single node, Tempest"
Also, maybe we should say "machine" here and above instead of node, since we use node later referring to graph nodes.